extensions/sql-migration/src/wizard/databaseSelectorPage.ts

```diff
@@ -212,18 +224,91 @@ export class DatabaseSelectorPage extends MigrationWizardPage {
 		// load unfiltered table list and pre-select list of databases saved in state
 		await this._filterTableList('', this.migrationStateModel._databasesForAssessment);
 
+		const xEventsDescription = this._view.modelBuilder.text()
+			.withProps({
+				value: constants.XEVENTS_ASSESSMENT_DESCRIPTION,
+				width: this.TABLE_WIDTH,
+				CSSStyles: { ...styles.BODY_CSS },
+				links: [{ text: constants.XEVENTS_ASSESSMENT_HELPLINK, url: 'aka.ms/sql-migration-xe-assess' }]
```

Comment:
is this url missing protocol?
https://aka.ma/sql-migration-xe-assess

Reply:
good catch, fixed!